packages/website/pages/faq.js

```diff
@@ -1,13 +1,102 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 
 import FAQPageData from '../content/pages/faq.json';
 import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
 import { initFloaterAnimations } from '../lib/floater-animations.js';
 import GeneralPageData from '../content/pages/general.json';
+import AccordionBlock from '../components/accordionblock/accordionblock';
+import Loading from '../components/loading/loading';
 
 export default function Home() {
+  const [faqSections, setFaqSections] = useState();
+  /** @type [any, null | any] */
+  const [propsToParseAsMarkdown, setPropsToParseAsMarkdown] = useState([]);
   const sections = FAQPageData.page_content;
   const animations = FAQPageData.floater_animations;
+  useEffect(() => {
+    fetch(`${process.env.NEXT_PUBLIC_NETLIFY_CMS_ENDPOINT || 'https://blog.web3.storage'}/api/partials/faq/faq`)
+      .then(async response => await response.text())
+      .then(text => {
+        const obj = JSON.parse(text);
+        const faqs = obj.props.partial.meta['faq-list'].reduce((sections, faq) => {
```

Comment:
You might want to accomodate a case where `obj` did not parse to the expected shape, because e.g. if `obj.props` is not an object, this will have an uncaught `TypeError`

Reply:
are we OK with the catch clause handling this error?

Reply:
sure, if there was a comment in the catch handler that that's what you expect to catch, I prob wouldn't have added my original comment

Reply:
comment added